### PR TITLE
Tickets/dm 28526

### PR DESCRIPTION
**Pull request by MelissaGraham**
Final updates prior to community advertising.

### CODE REVIEW

File: DMTN-049.tex [hunk 1]
```diff
@@ -123,14 +136,21 @@ \subsection{Federating a User-Generated Photo-$z$ Catalog}\label{ssec:time_ops_u
 
 \subsection{International Partnerships (In-Kind Contributions)}\label{ssec:time_inkind}
 
-All in-kind program contributions related to photo-$z$ will be subject to the processes outlined in this roadmap and will be included and integrated into the communal efforts to optimize the LSST photo-$z$.
-These contributions might include, e.g., individuals' expertise; scientific analyses; software algorithms, tools, or infrastructure; computational resources; datasets or observing time.
-These contributions might be directed to the Project (e.g., DM, commissioning team), and/or to the science community (e.g., a Science Collaboration, all data rights holders). 
+As the process for International Partnerships with Rubin Observatory via in-kind contributions evolved, photometric redshifts emerged as an area in which (1) multiple teams propose to provide different types of contributions and (2) optimizing the LSST scientific returns on these contributions requires the coordination across the Rubin community: the in-kind teams, Rubin staff, the Science Collaborations, and the broad community of data rights holders.
```
Comment (leannep) [inline DMTN-049.tex]: " requires coordination across the Rubin community:"

File: DMTN-049.tex [hunk 2]
```diff
@@ -123,14 +136,21 @@ \subsection{Federating a User-Generated Photo-$z$ Catalog}\label{ssec:time_ops_u
...
+Thus, the stages of this roadmap and the Photo-$z$ Coordination Group (\S~\ref{ssec:time_pzcoord}) were developed with the in-kind contributions in mind, to create a timeline and communication methods that will assist with the development and integration of the in-kind program contributions related to photo-$z$.
+
+At the time of Version 4 of this document, the proposed contributions included, e.g., individuals' expertise; scientific analyses; software algorithms, tools, or infrastructure; computational resources; datasets or observing time.
+The proposals specify the recipients for these contributions as either the Rubin Observatory project (e.g., DM, commissioning) or the science community (e.g., a Science Collaboration, all data rights holders).
+
+Here are a couple of examples to illustrate how in-kind teams would be involved in this roadmap.
+Participation level would vary between in-kind teams depending on their type of contribution.
```
Comment (leannep) [inline DMTN-049.tex]: "The participation level would ...." or "The level of participation would ..."

File: DMTN-049.tex [hunk 3]
```diff
@@ -123,14 +136,21 @@ \subsection{Federating a User-Generated Photo-$z$ Catalog}\label{ssec:time_ops_u
...
+In-kind teams proposing to contribute photo-$z$ related directable effort or software packages to Rubin Observatory might be helping DM to evaluate the "Letters of Recommendation", or to set up infrastructure to run the "Photo-z Validation Cooperative".
+In-kind teams proposing to contribute data or software to the Science Collaborations might be more focused on preparing "Letters of Recommendation", writing photo-$z$ algorithms, and then participating in the "Photo-z Validation Cooperative".
+
```
Comment (leannep) [inline DMTN-049.tex:152]: I'd put these 2 examples into an enumerated (or itemized) list

**Review by leannep — APPROVED** [review]

File: DMTN-049.tex (hunk 1 above)
Comment (MelissaGraham) [reply DMTN-049.tex]: fixed

File: DMTN-049.tex (hunk 2 above)
Comment (MelissaGraham) [reply DMTN-049.tex]: fixed

File: DMTN-049.tex (hunk 3 above)
Comment (MelissaGraham) [reply DMTN-049.tex:152]: went with an itemized list

**Pull request merged**